.circleci/config.yml

```diff
@@ -45,17 +45,9 @@ jobs:
       - checkout
 
       - run:
-          command: make linux darwin windows
+          command: make clean linux darwin windows
```

Comment:
Great question. There were two versions of this in the config.yml so I took the one that did the most. You are right that clean probably won't be needed, but I assumed it was there for a reason :)